### DISCUSSION TITLE
Host Retention Policy

### DISCUSSION THREAD

**Discussion by dkang-firmus** [CrowdStrike/falconpy#1205] [Q&A]
Hi Seniors,

I wanted to use API to check if Auto-Restore Active Hidden Hosts is enabled on a CID as well as checking the configurations of Host Retention Policies.

May I know if Falconpy supports this feature currently?

Thank you.

**Comment by jshcodes** [comment]
Hi @dkang-firmus -

Thank you for the question!

The API operations to support updates for host retention policies are not currently available.  When released, we will implement these operations in the SDK.

You __can__ view the currently assigned Host Retention policy via the Hosts API. ([GetDeviceDetailsV2](https://falconpy.io/Service-Collections/Hosts.html#getdevicedetailsv2))
